src/main/java/com/amazonaws/encryptionsdk/kms/AwsKmsMrkAwareMasterKey.java

```diff
@@ -90,7 +90,6 @@ private AwsKmsMrkAwareMasterKey(
           "AwsKmsMrkAwareMasterKey must be configured with an AWS KMS client.");
     }
 
-    /* Precondition: A provider is required. */
```

Comment:
Why are we removing these? This notation and associated CI check was a precursor to our spec/duvet work, but I wouldn't want to remove these unless we could have the spec/duvet fill the gap.

Reply:
I removed them here and added them to the v2 files because the conditions script will complain if there are duplicates.